navis/interfaces/h01.py

```diff
@@ -0,0 +1,40 @@
+from functools import lru_cache
+from . import cave_utils
+
+@lru_cache(None)
+def get_cave_client():
+    """Get caveclient for H01 dataset.
+    """
+    datastack = "h01_c3_flat"
+    server_address = "https://global.brain-wire-test.org/"
+    
+    return cave_utils.get_cave_client(datastack, server_address)
+
+def fetch_neurons(x, *, lod=2,
+                  with_synapses=True,
+                  datastack='h01_c3_flat',
+                  parallel=True,
+                  max_threads=4,
+                  **kwargs):
+    """Fetch neuron meshes.
```

Comment:
Not strictly necessary but it would be nice to have full docstrings for all functions in this module - you could just copy + adjust the docstrings from the `cave_utils` functions.